zk_toolbox/crates/zk_supervisor/src/commands/test/rust.rs

```diff
@@ -0,0 +1,41 @@
+use common::{cmd::Cmd, logger};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use super::args::rust::RustArgs;
+use crate::messages::{
+    MSG_CARGO_NEXTEST_MISSING_ERR, MSG_UNIT_TESTS_RUN_SUCCESS, MSG_USING_CARGO_NEXTEST,
+};
+
+pub fn run(shell: &Shell, args: RustArgs) -> anyhow::Result<()> {
```

Comment:
you have to setup the database as well. some of our tests are using the test-database. 
Please make sure, that you are not removing the ~production~ database